### DISCUSSION TITLE
QMenu - Limit width as per parent Input field

### DISCUSSION THREAD

**Discussion by ayushishah-easternts** [quasarframework/quasar#11662] [General - Components / Directives / etc] [answered]
**Describe the bug**
How to get width of q-menu same as parent input textfeld? I am trying to create a custom autocomplete type of component here. 
**Codepen/jsFiddle/Codesandbox (required or your issue may be closed)**
https://codepen.io/ayushibshah/pen/mdBOGxJ?editors=111

**To Reproduce**
1. Go to 'Search Inpuut'
2. Click on it
3. See the dropdown width is more than input search width, is there a way to bind that as per parent input field.

**Comment by pdanpdan** [comment]
Check QMenu docs - fit or cover props
And also check the docs for the example about limited with

**Reply by ayushishah-easternts** [reply]
I did try that still no luck @pdanpdan

**Comment by metalsadman** [comment, marked as answer]
use [Dom Util](https://quasar.dev/quasar-utils/dom-utils) to get width of your parent QInput, then set it as max-width of your QMenu's content.
https://codepen.io/metalsadman/pen/MWEJjPQ

**Reply by ayushishah-easternts** [reply]
Hey, thanks! I will try to work with this.

**Reply by metalsadman** [reply]
did some revision with comments, just choose which is more suitable for you.

**Reply by ayushishah-easternts** [reply]
@metalsadman It did work! Thank you so much.
